pkg/security/oauth/manager/configv2.go

```diff
@@ -0,0 +1,56 @@
+package manager
+
+import (
+	"fmt"
+	"net/url"
+	"time"
+
+	"github.com/plgd-dev/cloud/pkg/net/http/client"
+	"golang.org/x/oauth2/clientcredentials"
+)
+
+type ConfigV2 struct {
+	ClientID      string        `yaml:"clientID" json:"clientID"`
+	ClientSecret  string        `yaml:"clientSecret" json:"clientSecret"`
+	Scopes        []string      `yaml:"scopes" json:"scopes"`
+	TokenURL      string        `yaml:"tokenURL" json:"tokenURL"`
+	Audience      string        `yaml:"audience" json:"audience"`
+	HTTP          client.Config `yaml:"http" json:"http"`
+	TickFrequency time.Duration `yaml:"tickFrequency" json:"tickFrequency"`
```

Comment:
Tick won't tell me what is ticking. Describe the purpose of this and not how often is it ticking.